packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -156,16 +168,25 @@ contract Reserve is
     emit TobinTaxReserveRatioSet(value);
   }
 
-  /**
-   * @notice Set the ratio of reserve that is spendable per day.
-   * @param ratio Spending ratio as unwrapped Fraction.
-   */
   function setDailySpendingRatio(uint256 ratio) public onlyOwner {
     spendingRatio = FixidityLib.wrap(ratio);
     require(spendingRatio.lte(FixidityLib.fixed1()), "spending ratio cannot be larger than 1");
     emit DailySpendingRatioSet(ratio);
   }
 
+  /**
+   * @notice Set the ratio of reserve for a given ERC20 token that is spendable per day.
+   * @param erc20SpendingRatio Spending ratio as unwrapped Fraction.
+   */
+  function setDailySpendingRatioForErc20Tokens(
+    address erc20TokenAddress,
```

Comment:
If you add this function to the initialize function you empty case

Reply:
done!